lib/pagseguro/payment_request.rb

```diff
@@ -57,6 +57,13 @@ class PaymentRequest
     # The token that identifies the request. Defaults to PagSeguro.token
     attr_accessor :token
 
+    # Determines which PagSeguro API version the request will use. Defaults to v2
+    attr_writer :api_version
+
+    # The default PagSeguro API version
+    def api_version
+      @api_version ||= 'v2'
+    end
```

Comment:
why do it need an `attr_writer` for api_version?

the `api_version` method can be implemented like this:

```ruby
private

def api_version
  'v2'
end
```

following this way, we are not explicitly allowing an overwrite of api version. and we don't need to store the value in an instance var neither.